.github/workflows/validate.yml

```diff
@@ -1,8 +1,7 @@
-name: ValidateTrialJsonSchema
+name: Validate Trial Config Files
```

Comment:
I think this is a good time to move the validation step to the primary workflow since we don't want to run it if the validation fails.
Also, I think we an skip the hassle of figuring which definition changed and just validate all of them.

Reply:
Agree, done